app/routes/goal_routes.py

```diff
@@ -1 +1,89 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, Response, request, abort, make_response
```

Comment:
Response, abort, and make_response are imported but never accessed so should be removed.